### DISCUSSION TITLE
[10/1/21] Issues with Vercel dashboard

### DISCUSSION THREAD

**Discussion by jinan159** [vercel/vercel#6793] [Help] [answered]
Vercel github login is not working now.. 
Is this just on me?
And my project's production build is pending 15min.

![Screenshot 2021-10-01 23 33 07](https://user-images.githubusercontent.com/45728407/135637903-a7b588da-3f47-41c1-a27a-c9e34ee87631.png)

**Comment by jorgemndoza** [comment]
Vercel service is down for me too. My apps and pages are not loading.

**Comment by aplchian** [comment]
Vercel is down for my team as well.

**Comment by bscaspar** [comment]
For me as well. And some customers have seen ERR_CERT_DATE_INVALID issues.

**Comment by RobEasthope** [comment]
Vercel Dashboard is stuck loading and the console log is screaming error messages. Builds are slow but appear to be completing but Github isn't updating to reflect the build status

**Comment by jorgemndoza** [comment]
seems like they are aware of the issue.
https://www.vercel-status.com/

**Comment by leerob** [comment, marked as answer]
Sorry about this! We are investigating the issue and will post updates on the status page: https://www.vercel-status.com/

Edit: This has been mitigated.
